packages/nuxt/src/head/module.ts

```diff
@@ -68,9 +75,23 @@ export default import.meta.server ? [CapoPlugin({ track: true })] : [];`
       },
     })
 
+    addTemplate({
+      filename: 'unhead.config.mjs',
+      getContents (ctx) {
+        const renderSSRHeadOptions = defu(options.renderSSRHeadOptions, {
+          omitLineBreaks: !(ctx.nuxt.options.dev || ctx.nuxt.options.test),
+        })
```

Comment:
I think this can be moved up into `defaults` function above `setup` and it will be pre-normalised for us...

Reply:
OK! I've updated it 😁